src/HeroSection.php

```diff
@@ -169,45 +134,45 @@ public function title() {
 		if ( class_exists( 'WooCommerce' ) && is_shop() ) {
 
 			genesis_markup(
-				array(
+				[
```

Comment:
You'll have to increase the required PHP version to 5.4 for short array syntax.

Reply:
Good spotting. Thought that was 5.3 for some reason. I’ve been using it in the theme config for a while now.